src/main/java/emissary/admin/Startup.java

```diff
@@ -274,7 +274,7 @@ protected boolean localDirectorySetup(final Map<String, String> localDirectories
             if (KeyManipulator.isLocalTo(thePlaceLocation, "http://" + this.node.getNodeName() + ":" + this.node.getNodePort() + "/StartupEngine")) {
                 logger.info("Doing local startup for directory {} ", thePlaceLocation);
                 final String thePlaceClassStr = PlaceStarter.getClassString(thePlaceLocation);
-                final IServiceProviderPlace p = PlaceStarter.createPlace(thePlaceLocation, null, thePlaceClassStr, null);
+                final IServiceProviderPlace p = PlaceStarter.createPlace(thePlaceLocation, null, thePlaceClassStr, null, new EmissaryNode());
```

Comment:
Does `EmissaryNode` need to be created for each iteration? If not, recommend moving to outside of the for loop.